src/interfaces/IMorpho.sol

```diff
@@ -298,3 +280,41 @@ interface IMorpho {
     /// @notice Returns the data stored on the different `slots`.
     function extSloads(bytes32[] memory slots) external view returns (bytes32[] memory);
 }
+
+/// @title IMorphoStaticTyping
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @dev This interface is inherited by Morpho so that function signatures are checked by the compiler.
+/// @dev Consider using the IMorpho interface instead of this one.
+interface IMorphoStaticTyping is IMorphoBase {
+    /// @notice The state of the position of `user` on the market corresponding to `id`.
+    function position(Id id, address user)
+        external
+        view
+        returns (uint256 supplyShares, uint128 borrowShares, uint128 collateral);
+
+    /// @notice The state of the market corresponding to `id`.
+    function market(Id id)
+        external
+        view
+        returns (
+            uint128 totalSupplyAssets,
+            uint128 totalSupplyShares,
+            uint128 totalBorrowAssets,
+            uint128 totalBorrowShares,
+            uint128 lastUpdate,
+            uint128 fee
+        );
+}
+
+/// @title IMorpho
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
```

Comment:
We didn't add this in `metamorpho`

Should we?

Reply:
I don't know, we are kind of breaking one of our convention because we don't have one library/interface/contract per file. I think it's fine because we want those to be "internal", in the sense that they are not supposed to be used outside of this file. Because of this, now that I think about it, I would just add this header to `IMorpho`, the interface that is actually meant to be used